spec/support/api/helpers.rb

```diff
@@ -40,6 +40,13 @@ def stub_api_action_role(collection, action_type, method, action, identifier)
           allow(::Api::ApiConfig.collections[collection][action_type]).to receive(method) { updated_method }
         end
 
+        def collection_action_classed_identifier(type, action, method, klass)
```

Comment:
I don't believe this needs the action actually but AB said to use this signature n before this gets merged it'd be great to have someone else's eyes on it

Reply:
I think `action` will be needed in the future when we need to use this to bulk delete or run other operations that are not as simple as single instance CRUD.  Is that correct @abellotti ?

Reply:
yes, just discussed with @d-m-u, we need the action:

```
::Api::ApiConfig.collections[type][:collection_actions][method].detect { |spec| spec[:name] == action }[:identifiers].detect { |spec| spec[:klass] == klass.to_s }
```